src/server/admin/permissions/controllers/create/permission-details.js

```diff
@@ -39,7 +39,7 @@ const createPermissionDetailsController = {
 
     if (!validationResult.error) {
       try {
-        const { payload: createScopePayload } = await createScope(request, {
+        const createdScope = await createScope(request, {
```

Comment:
is this working? `payload` is provided back from `fetchJson` on all responses returned. I would expect this (unless fetchJson has changed) to be `createdScope.payload.scopeId`

Reply:
Yes, has been changed in `createScope`